lablib/generators/ocio_config.py

```diff
@@ -186,6 +199,16 @@ def clear_vars(self):
         """Clear the environment variables."""
         self._vars = {}
 
+    def clear_search_paths(self):
```

Comment:
not really sure if we need functions for clearing internal lists as it's really just one line.

it's really just a cosmetic thing imho. we can leave but if so... let's get some docstrings in there :)